### PR TITLE
refactor: [v6] rm Tag margin style

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [x] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

- #52115

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | refactor: rm Tag margin style |
| 🇨🇳 Chinese | refactor: rm Tag margin style |

### CODE REVIEW

**Comment by zombieJ** [comment]
我记得之前讨论是去掉后改成一个 `token` 出来

**Comment by li-jia-nan** [comment]
> 我记得之前讨论是去掉后改成一个 `token` 出来

这个要怎么命名呢？

**Comment by MadCcc** [comment]
> 我记得之前讨论是去掉后改成一个 `token` 出来

不加了吧，用 style 就行了

**Comment by li-jia-nan** [comment]
Visual Diff 貌似又卡住了

File: components/tag/demo/control.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const App: React.FC = () => {
   };
 
   return (
-    <Flex gap="4px 0" wrap>
+    <Flex gap="small" align="center" wrap>
```
Comment (zombieJ) [inline components/tag/demo/control.tsx:74]: 拆到 master 里？免得以后合并老是要解决冲突

**Review by zombieJ — DISMISSED** [review]

**Review by zombieJ — DISMISSED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**